src/PlanetData.js

```diff
@@ -43,18 +48,21 @@ PlanetData.prototype = {
         red = c.r * d.x;
         green = c.g * d.x;
         blue = c.b * d.x;
-        this.mapData.pixels[y * this.mapData.width + x] = Phaser.Color.packPixel(red, green, blue, alpha);
+        i = 4 * (y * this.width + x);
+        this.mapData.data[i] = red;
+        this.mapData.data[i + 1] = green;
+        this.mapData.data[i + 2] = blue;
+        this.mapData.data[i + 3] = alpha;

```

Comment:
Sorry, this is a side affect from working with the Javascript canvas object. For speed reasons they store the internal variable in a Uint8 array. And since we are working directly on a canvas now we can't change that. The only alternative would be to store it in a 2D array and then have another loop after to pass the values from the 2D array directly into the canvas buffer.